File: UtilitiesVB/Solution Interfaces/IOlObjects.vb
Imports Microsoft.Office.Interop.Outlook

Public Interface IOlObjects
    ReadOnly Property App As Application
    ReadOnly Property EmailRootPath As String
    ReadOnly Property ArchiveRootPath As String
    ReadOnly Property Inbox As Folder
    ReadOnly Property NamespaceMAPI As [NameSpace]
    ReadOnly Property OlEmailRoot As Folder
    ReadOnly Property OlReminders As Reminders
    ReadOnly Property ToDoFolder As Folder
    Property ShowInConversations As Boolean
    Property MovedMails_Stack As StackObjectVB
    ReadOnly Property View_Wide As String
End Interface

File: UtilitiesVB/Solution Interfaces/IOlObjects.cs
using Microsoft.Office.Interop.Outlook;

namespace UtilitiesVB
{

    public interface IOlObjects
    {
        Application App { get; }
        string EmailRootPath { get; }
        string ArchiveRootPath { get; }
        string UserEmailAddress { get; }
        Folder Inbox { get; }
        NameSpace NamespaceMAPI { get; }
        Folder OlEmailRoot { get; }
        Reminders OlReminders { get; }
        Folder ToDoFolder { get; }
        bool ShowInConversations { get; set; }
        StackObjectVB MovedMails_Stack { get; set; }
        string View_Wide { get; }
    }
}